mne/viz/topomap.py

```diff
@@ -1037,7 +1037,7 @@ def plot_evoked_topomap(evoked, times=None, ch_type='mag', layout=None,
     if colorbar:
         cax = plt.subplot(1, n + 1, n + 1)
         plt.colorbar(images[-1], ax=cax, cax=cax, ticks=[vmin, 0, vmax],
-                     format=format)
+                     format=fmt)
```

Comment:
Alex, `format` is an inbuilt function.

2015-02-15 18:03 GMT+01:00 Alexandre Gramfort <notifications@github.com>:

> In mne/viz/topomap.py
> <https://github.com/mne-tools/mne-python/pull/1800#discussion_r24724045>:
>
> > @@ -1037,7 +1037,7 @@ def plot_evoked_topomap(evoked, times=None, ch_type='mag', layout=None,
> >      if colorbar:
> >          cax = plt.subplot(1, n + 1, n + 1)
> >          plt.colorbar(images[-1], ax=cax, cax=cax, ticks=[vmin, 0, vmax],
> > -                     format=format)
> > +                     format=fmt)
>
> ok but it's a method of str type so there is not really any conflict
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/1800/files#r24724045>.
>